_Mod/LWM.DeepStorage/Patches/Compatibility_StockpileForDisaster.xml

```diff
@@ -4,38 +4,40 @@
      Adding patch as seen in StockpileForDisaster's Patches/otherMods_Patch.xml
      -->
 <Patch>
-  <Operation Class="PatchOperationFindMod">
-    <mods>
-      <li>[1.0]-StockpileForDisaster</li>
-    </mods>
-    <nomatch Class="LWM.DeepStorage.PatchMessage">
-      <message>WTF</message>
-    </nomatch>
-    <match Class="PatchOperationSequence">
-      <operations>
-	<li Class="PatchOperationAdd">
-	  <xpath>/Defs/ThingDef[defName="LWM_DeepStorage"]/inspectorTabs</xpath>
-	  <value><li>StockpileForDisaster.ITab_Restrict</li></value>
-	</li>
-	<li Class="PatchOperationAdd">
-	  <xpath>/Defs/ThingDef[defName="LWM_DeepStorage"]/comps</xpath>
-	  <value>
-            <li Class="StockpileForDisaster.EntityListCompDef">
-              <compClass>StockpileForDisaster.EntityListComp</compClass>
-              <listType>TypeOnlyRecieve</listType>
-              <nodeDef>RecieveRoot</nodeDef>
-              <canDisplayCategorys>
-		<li>CategoryMale</li>
-		<li>CategoryFemale</li>
-		<li>CategoryPets</li>
-              </canDisplayCategorys>
-            </li>
-	  </value>
-	</li>
-        <li Class="LWM.DeepStorage.PatchMessage">
-          <message>LWM Deep Storage: activated compatibility patch for [1.0]-StockpileForDisaster</message>
-        </li>
-      </operations>
-    </match>
-  </Operation>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>[1.0]-StockpileForDisaster</li>
+		</mods>
+		<nomatch Class="LWM.DeepStorage.PatchMessage">
+			<message>WTF</message>
+		</nomatch>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="LWM_DeepStorage"]/inspectorTabs</xpath>
```

Comment:
What am I writing?

Reply:
I'm not familiar with StockpileForDisaster, so all I did was just cleanup your indents.